pkg/controller/podautoscaler/horizontal.go

```diff
@@ -39,6 +39,8 @@ const (
 	// TODO: make it a flag or HPA spec element.
 	tolerance = 0.1
 
+	defaultTargetCPUUtilizationPercentage = 80
```

Comment:
Ah. I guess I still read "if not specified the default autoscaling policy will be used" and my first thought is "where do I set the default autoscaling policy?"